resources/public/api/conf/1.0/checkpoints_create.yaml

```diff
@@ -0,0 +1,53 @@
+summary: Create a checkpoint
+description: |
+  This endpoint allows a developer to set up a checkpoint for the stateful data in the 
+  sandbox environment for a particular NINO. A National insurance number must be provided.
```

Comment:
Minor wording thing, but should we be consistent with using the phrase `NINO` or `National insurance number` (does referencing them different in the same description imply they are different?)?

Reply:
Probably should use "National insurance number". I will fix.